feature/platform/transceiver/tests/zr_input_output_power_test/zr_input_output_power_test.go

```diff
@@ -0,0 +1,416 @@
+package zr_input_output_power_test
+
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package topology_test configures just the ports on DUT and ATE,
```

Comment:
This isn't the name of the package and a description of the test belongs in the README file anyway.

Reply:
Addressed in the latest commit.